### DISCUSSION TITLE
Ability to snooze co-pilot for short amount of time

### DISCUSSION THREAD

**Discussion by oleg-slapdash** [community/community#52094] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Sometimes I am writing a comment/code and I have content in mind and I need time to focus and express it. But co-pilot keeps distracting with suggestions. There should be a way to stop it for a short period of time.

**Comment by frntn** [comment]
Well. It seems that they listened or actually had the same issue :D 

<img width="242" height="339" alt="image" src="https://github.com/user-attachments/assets/5b246bac-9e0d-4283-90b0-af0fc749d9f9" />
